src/main/java/org/sonar/plugins/stash/issue/collector/SonarQubeCollector.java

```diff
@@ -1,31 +1,14 @@
 package org.sonar.plugins.stash.issue.collector;
 
 import java.io.File;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.io.File'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)